docs/development.md

```diff
@@ -119,6 +119,60 @@ recommended, since it requires a lot of effort to manually install all the tool
 natively on your machine and keeping them up to date to the exact same version
 specified in Docker.
 
+## rootless Docker
+
+In order to run Docker without root privileges, follow the guide at
+https://docs.docker.com/engine/security/rootless/ .
+
+Below is a quick summary of the relevant steps.
+
+1. If you have an existing version of Docker running as root, uninstall that
+   first.
+
+   ```bash
+   sudo systemctl disable --now docker.service docker.socket
+   sudo apt remove docker-ce docker-engine docker-runc docker-containerd
+   ```
+
+1. Install `uidmap`.
+
+   ```bash
+   sudo apt install uidmap
+   ```
+
+1. Add a range of subids for the current user.
+
+   ```bash
+   sudo usermod --add-subuids 500000-565535 --add-subgids 500000-565535 $USER
+   ```
+
+1. Download the install script for rootless docker, and run it as the current
+   user.
+
+   ```bash
+   curl -fSSL https://get.docker.com/rootless > $HOME/rootless
+   sh $HOME/rootless
+   ```
+
```

Comment:
Maybe add: You can check the context with `docker context ls`.

Reply:
Good point, done.